src/Online_Bookstore/Dashboard/BookDetail.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+import useFetch from "../CustomHooks/useFetch";
+import BookDetailView from "./BookDetailView";
+import {handleSubmit} from './SetReviewInLocalStorage'
+const BookDetail = () => {
+  const [comment, setComment] = useState("");
+  const [rating, setRating] = useState(0);
```

Comment:
Combine these two states together for better readability and clean code practice. Study the DRY concept in overall programming as well as React.

```
const [review, setReview] = useState({
     comment: '',
     rating: 0
})
```

@Poorvi-3pillar